<!-- -->
<launch>
    <node type="radar_broadcaster"
          pkg="adeye"
          name="radar_broadcaster"
          respawn="true"/>
</launch>
